packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -414,12 +421,25 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
 
   private _onScroll = () => {
     const { contentContainer } = this;
-
     if (contentContainer) {
       this._stickies.forEach((sticky: Sticky) => {
         sticky.syncScroll(contentContainer);
       });
     }
+    const { stickyTopHeight, stickyAboveClassName } = this.state;
+    if (this.props.onScrollStickyAboveClassName !== undefined && stickyAboveClassName !== undefined && stickyTopHeight > 0) {
+      const scrollPosition = this.getScrollPosition();
+      const isOnScrollClassApplied = stickyAboveClassName.includes(this.props.onScrollStickyAboveClassName);
```

Comment:
equality '===' check instead of 'contains'? Thats the only valid value we are assigning anyways.

Reply:
0 then false